submissions/tests/test_override_score.py

```diff
@@ -0,0 +1,115 @@
+"""
+Test override scores.
+"""
+
+from mock import patch
+from django.test import TestCase
+from django.core.cache import cache
+from django.db import DatabaseError
+from submissions import api as sub_api
+from submissions.models import Score
+
+
+class TestOverrideScore(TestCase):
+    """
+    Test overriding scores for a specific student on a specific problem.
+    """
+
+    STUDENT_ITEM = {
+        'student_id': 'Test student',
+        'course_id': 'Test course',
+        'item_id': 'Test item',
+        'item_type': 'Test item type',
+    }
+
+    def setUp(self):
+        """
+        Clear the cache.
+        """
+        cache.clear()
+
+    def test_override_with_no_score(self):
+
+        sub_api.score_override(
+            self.STUDENT_ITEM,
+            10,
+            8,
+        )
+
+        self.assertEqual(sub_api.get_score(self.STUDENT_ITEM)['points_earned'], 8)
+        self.assertEqual(sub_api.get_score(self.STUDENT_ITEM)['points_possible'], 10)
+
+    def test_override_with_one_score(self):
```

Comment:
I saw that test too for reset. Since we're always getting the latest it doesn't seem necessary for overrides.